src/org/opendatakit/briefcase/ui/PullTransferPanel.java

```diff
@@ -284,7 +284,7 @@ public void focusLost(FocusEvent e) {
     formTransferTable.setSourceSelected(true);
     JScrollPane scrollPane = new JScrollPane(formTransferTable);
 
-    sourceConfPanel = new SourceConfigurationPanel();
+    SourceConfigurationPanel sourceConfPanel = new SourceConfigurationPanel(PULL);
```

Comment:
Better: `SourceConfigurationPanel.pull()`

Reply:
Ok onto it :+1: